include/aws/common/config.h.in

```diff
@@ -19,5 +19,6 @@
 #cmakedefine AWS_HAVE_EXECINFO
 #cmakedefine AWS_HAVE_WINAPI_DESKTOP
 #cmakedefine AWS_HAVE_LINUX_IF_LINK_H
+#cmakedefine INTEL_NO_ITTNOTIFY_API
```

Comment:
we talked, it's probably simpler to have AwsCFlags.c add INTEL_NO_ITTNOTIFY_API to AWS_C_DEFINES_PRIVATE

Reply:
Thanks, fixed.